docs/topics/commands.rst

```diff
@@ -461,6 +461,9 @@ Supported options:
 * ``--meta`` or ``-m``: additional request meta that will be passed to the callback 
   request. This must be a valid json string. Example: --meta='{"foo" : "bar"}'
 
+* ``--cb_kwargs``: additional keyword arguments that will be passed to the callback.
```

Comment:
I wonder if we should apply a CLI-specific naming convention to the parameter there. I’m used to hyphens (``--cb-kwargs``), although skipping separators altogether seems to be the current convention (``--cbkwargs``).

I also wonder if we should even deviate from the Python parameter name here, and use the whole word (‘callback’).

Just thinking out loud, though. I’m don’t oppose merging with `--cb_kwargs`.

Reply:
Makes sense, `--cbkwargs` would be coherent with the current `--nolinks`, `--noitems`, `--nocolour` options.